### PR TITLE
Add collective.taskqueue buildouts.

### PR DESCRIPTION
**Pull request by jone**
Provide standard taskqueue queue configurations for production and for development, including a local volatile queue and a redis queue configuration each.

Requires https://github.com/4teamwork/ftw-buildouts/pull/45 (Redis)

/cc @lukasgraf @maethu @buchi

### CODE REVIEW

**Comment by jone** [comment]
@4teamwork/dev-plone With this PR we should stop configuring taskqueues, e.g. for bumblebee, in the project buildouts but include these configurations.

Since the normal queue is volatile, meaning that the queue gets lost when rebooting instances, we should start installing redis.

Background:
I have tried to replace the volatile in-memory queue (`Queue.Queue`) with a `zc.queue.Queue`, but was not successful yet because of transaction / connection issues with the thread. Therefore I tend to just using redis as queue and configuring it to persist, as set up in #45.

**Comment by jone** [comment]
Updated after #45 was merged.

File: taskqueue/redis-development.cfg [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# collective.taskqueue configuration, configuring a redis taks queue
```
Comment (lukasgraf) [inline taskqueue/redis-development.cfg]: `taks` -> `task`

File: taskqueue/redis-development.cfg [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Requires:
+# - development.cfg
+# - redis/development.cfg
+
+
+[instance]
+# Load redis extras of collective.taskqueue:
+eggs += collective.taskqueue [redis]
+
+zope-conf-additional =
```
Comment (lukasgraf) [inline taskqueue/redis-development.cfg]: Shouldn't this be `+=`? (Same in `redis-production.cfg`)

**Comment by jone** [comment]
@lukasgraf updated

**Comment by lukasgraf** [comment]
👍

**Pull request merged**